crates/router/src/connector/wise/transformers.rs

```diff
@@ -0,0 +1,617 @@
+                        field_name: "profile",
+                    }
+                })?;
+                Ok(Self {
+                    target_account,
+                    quote_uuid,
+                    customer_transaction_id: request.payout_id,
+                    details: wise_transfer_details,
+                })
+            }
+            storage_enums::PayoutType::Card => Err(errors::ConnectorError::NotSupported {
+                message: "Card payout fulfillment is not supported".to_string(),
+                connector: "Wise",
+                payment_experience: "".to_string(),
+            })?,
+        }
+    }
+}
+
+// Payouts transfer creation response
+#[cfg(feature = "payouts")]
+impl<F> TryFrom<types::PayoutsResponseRouterData<F, WisePayoutResponse>>
+    for types::PayoutsRouterData<F>
+{
+    type Error = Error;
+    fn try_from(
+        item: types::PayoutsResponseRouterData<F, WisePayoutResponse>,
+    ) -> Result<Self, Self::Error> {
+        let response: WisePayoutResponse = item.response;
+        let status = match storage_enums::PayoutStatus::foreign_from(response.status) {
+            storage_enums::PayoutStatus::Cancelled => storage_enums::PayoutStatus::Cancelled,
+            _ => storage_enums::PayoutStatus::RequiresFulfillment,
+        };
+
+        Ok(Self {
+            response: Ok(types::PayoutsResponseData {
+                status: Some(status),
+                connector_payout_id: response.id.to_string(),
+                payout_eligible: None,
+            }),
+            ..item.data
+        })
+    }
+}
+
+// Payouts fulfill request transform
+#[cfg(feature = "payouts")]
+impl<F> TryFrom<&types::PayoutsRouterData<F>> for WisePayoutFulfillRequest {
+    type Error = Error;
+    fn try_from(item: &types::PayoutsRouterData<F>) -> Result<Self, Self::Error> {
+        let request = item.request.to_owned();
+        match request.payout_type.to_owned() {
+            storage_enums::PayoutType::Bank => Ok(Self {
+                fund_type: FundType::default(),
+            }),
+            storage_enums::PayoutType::Card => Err(errors::ConnectorError::NotSupported {
+                message: "Card payout fulfillment is not supported".to_string(),
+                connector: "Wise",
+                payment_experience: "".to_string(),
+            })?,
+        }
+    }
+}
+
+// Payouts fulfill response transform
+#[cfg(feature = "payouts")]
+impl<F> TryFrom<types::PayoutsResponseRouterData<F, WiseFulfillResponse>>
+    for types::PayoutsRouterData<F>
+{
+    type Error = Error;
+    fn try_from(
+        item: types::PayoutsResponseRouterData<F, WiseFulfillResponse>,
+    ) -> Result<Self, Self::Error> {
+        let response: WiseFulfillResponse = item.response;
+
+        Ok(Self {
+            response: Ok(types::PayoutsResponseData {
+                status: Some(storage_enums::PayoutStatus::foreign_from(response.status)),
+                connector_payout_id: "".to_string(),
```

Comment:
we won't get payout_id in fulfill_response?

Reply:
No, there is no fulfill ID in the response

reference - https://api-docs.transferwise.com/api-reference/transfer#fund